modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -654,7 +655,7 @@ namespace gapi {
 // A type-erased form of network parameters.
 // Similar to how a type-erased GKernel is represented and used.
 /// @private
-struct GAPI_EXPORTS GNetParam {
+struct GAPI_EXPORTS_W_SIMPLE GNetParam {
```

Comment:
Is `GNetParam` wrapped now? Is this a functional intersection with `PyParams`, if yes?

Reply:
Don't get your question

Reply:
Discussed locally

Reply:
Discussed locally.